contrib/rspamd/template.py

```diff
@@ -0,0 +1,62 @@
+pkgname = "rspamd"
+pkgver = "3.7.5"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCONFDIR=/etc/rspamd",
+    "-DENABLE_FASTTEXT=ON",
+    "-DENABLE_HYPERSCAN=OFF",
+    "-DENABLE_LUAJIT=OFF",
+    "-DENABLE_URI_INCLUDE=ON",
+    "-DRSPAMD_GROUP=_rspamd",
+    "-DRSPAMD_USER=_rspamd",
+    "-DSYSTEM_FMT=ON",
+    "-DSYSTEM_XXHASH=ON",
+    "-DSYSTEM_ZSTD=ON",
+    "-D_CAN_RUN=0",
+    "-DHAVE_ATOMIC_BUILTINS_EXITCODE=0",
+]
+make_build_args = ["--target", "all", "check"]
+hostmakedepends = ["cmake", "ninja", "perl", "pkgconf", "ragel"]
+makedepends = [
+    "elfutils-devel",
+    "fasttext-devel",
+    "fmt-devel",
+    "glib-devel",
+    "icu-devel",
+    "libsodium-devel",
+    "libunwind-devel",
+    "lua5.4-devel",
+    "openssl-devel",
+    "pcre2-devel",
+    "snowball-devel",
+    "sqlite-devel",
+    "xxhash-devel",
+    "zstd-devel",
+]
```

Comment:
didn't you intend to use vectorscan as a dep here

Reply:
ah yes, must have missed that!